### PR TITLE
Add french sql book

### PR DESCRIPTION
**Pull request by x0rld**
## What does this PR do?
Add resource

### Why is this valuable (or not)?
it's a well known SQL course

### How do we know it's really free?
the PDF is there 
https://sql.sh/ressources/cours-sql-sh-.pdf

### For book lists, is it a book? For course lists, is it a course? etc.
it's a book available in PDF, Epub and as the website
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-fr.md [hunk 1]
```diff
@@ -323,6 +323,7 @@
 ### SQL
 
 * [Cours complet pour apprendre les différents types de bases de données et le langage SQL](https://sgbd.developpez.com/tutoriels/cours-complet-bdd-sql/) - Jacques Le Maitre
+* [Cours de SQL sql.sh](https://sql.sh) - (en ligne, PDF, EPUB)
```
Comment (eshellman) [inline books/free-programming-books-fr.md:326]: Don't invent titles! Use the title given by the resource.

**Comment by x0rld** [comment]
image
I put the same title as the pdf said

File: books/free-programming-books-fr.md [hunk 2]
```diff
@@ -323,6 +323,7 @@
 ### SQL
 
 * [Cours complet pour apprendre les différents types de bases de données et le langage SQL](https://sgbd.developpez.com/tutoriels/cours-complet-bdd-sql/) - Jacques Le Maitre
+* [Cours de SQL base du langage SQL et des bases de données](https://sql.sh) - (en ligne, PDF, EPUB)
```
Comment (eshellman) [inline books/free-programming-books-fr.md:326]:
```suggestion
* [Cours de SQL base du langage SQL et des bases de données](https://sql.sh) - Tony Archambeau
```
Please add author names where appropriate.
don't need list of filetypes if everything is accessible from the linked page

Comment (x0rld) [reply books/free-programming-books-fr.md]: it's done

**Comment by x0rld** [comment]
@eshellman all is fine now ?

**Pull request merged**